package.json

```diff
@@ -1,102 +1,107 @@
+    "ember-changeset": "^2.2.4",
+    "ember-changeset-validations": "^2.2.1",
+    "ember-cli": "~3.12.0",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^6.6.0",
-    "ember-cli-chai": "^0.5.0",
-    "ember-cli-chart": "^3.4.0",
-    "ember-cli-code-coverage": "^0.4.2",
-    "ember-cli-dependency-checker": "^3.0.0",
-    "ember-cli-eslint": "^5.0.0",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^0.4.4",
-    "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-mirage": "^0.3.4",
-    "ember-cli-mocha": "^0.15.0",
+    "ember-cli-babel": "^7.13.2",
+    "ember-cli-chart": "^3.5.0",
+    "ember-cli-code-coverage": "^1.0.0-beta.8",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-htmlbars": "^4.2.0",
+    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-mirage": "^1.1.6",
     "ember-cli-moment-shim": "^3.5.0",
-    "ember-cli-nouislider": "^1.0.0",
-    "ember-cli-postcss": "^4.0.0",
+    "ember-cli-nouislider": "^1.1.0",
     "ember-cli-release": "^1.0.0-beta.2",
-    "ember-cli-sass": "^7.0.0",
-    "ember-cli-shims": "^1.1.0",
-    "ember-cli-sri": "^2.1.0",
+    "ember-cli-sass": "^10.0.0",
+    "ember-cli-shims": "^1.2.0",
+    "ember-cli-sri": "^2.1.1",
     "ember-cli-test-loader": "^2.1.0",
     "ember-cli-uglify-es": "^1.0.4",
-    "ember-component-css": "^0.6.4",
-    "ember-composable-helpers": "^2.0.3",
-    "ember-concurrency": "^0.8.12",
-    "ember-data": "~2.18.0",
-    "ember-export-application-global": "^2.0.0",
-    "ember-fetch": "^5.1.1",
-    "ember-font-awesome": "^3.0.5",
-    "ember-load-initializers": "^1.0.0",
-    "ember-math-helpers": "^2.2.3",
+    "ember-cli-update": "^0.47.1",
+    "ember-component-css": "^0.6.9",
+    "ember-composable-helpers": "^3.0.2",
+    "ember-concurrency": "^1.1.3",
+    "ember-data": "~3.12.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-fetch": "^7.0.0",
+    "ember-font-awesome": "^3.1.1",
+    "ember-load-initializers": "^2.1.1",
+    "ember-math-helpers": "^2.7.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-mocha": "^0.12.0",
-    "ember-moment": "^7.4.1",
+    "ember-moment": "^7.6.0",
     "ember-native-dom-helpers": "^0.6.1",
-    "ember-notify": "^5.2.1",
+    "ember-notify": "^6.0.0",
     "ember-one-way-controls": "^3.0.1",
-    "ember-pad": "^1.2.2",
-    "ember-parachute": "^0.3.5",
-    "ember-power-calendar": "^0.7.3",
-    "ember-power-select": "^1.9.11",
-    "ember-resolver": "^5.0.1",
-    "ember-responsive": "^2.0.5",
-    "ember-route-action-helper": "^2.0.6",
-    "ember-scrollable": "^0.5.0",
-    "ember-simple-auth": "^1.4.0",
-    "ember-site-tour": "^1.1.0",
-    "ember-source": "~2.18.0",
-    "ember-test-selectors": "^1.0.0",
-    "ember-tether": "^1.0.0-beta.0",
-    "ember-truth-helpers": "^2.0.0",
-    "ember-validated-form": "^0.6.0",
-    "ember-wormhole": "^0.5.2",
+    "ember-pad": "^1.2.3",
```

Comment:
Can be replaced by `String.padEnd` or `String.padStart`

Reply:
I keep this one as the project also uses `padStartTpl` and the `padStart` function is often used on numbers. As having `padStart(number, 2)` is - IMO - nicer than `String.prototype.padStart(number, 2)` I decided to leave this as-is.

Reply:
Hm, don't really like that dependency since it's unmaintained - however, I see your point.. Let's leave it as it is

Reply:
Ok. I didn't catch that it is not maintained anymore. I'll look with topaxi. Maybe he'll transfer the package to Adfinis.